css/style.css

```diff
@@ -0,0 +1,144 @@
+*{
+    
+}
+body{
+    background-color: #eee;
+    font-family: 'Lato', sans-serif
+}
+.header{
+    display: flex;
+    padding-top: 25px;
+    background-color: #eee;
+    
+}
+.headerlink{
+    padding-left: 500px;
+    display: flex;
+    gap: 150px;
+}
+a{
+    color: black;
+    text-decoration: none;
+}
+.original{
+    font-size: 20px;
+    display: flex;
+    gap: 30px;
+}
+
+img{
+    width: 400px;
+}
+
+.hero{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 200px;
+    margin-top: 50px;
+    
+}
+.hero > h2 {
+    margin-bottom: 20px;
+    word-wrap: break-word;
+  }
+
+
+.placeholder{
+    color: gray;
+    background-color: white;
+    border: 1px solid black;
+    max-width: 275px;
+    text-align: left;
+    width: 100%;
+    padding: 5px;
+}
+.container{
+    max-width: 1000px;
+         width: 100%;
+        margin: 0 auto;
+    }
+.get{
+    width: 150px;
+       font-size: 20px;
+       font-weight: bold;
+        height: 30px ;
+        margin:15px 0;
+        border: 0;
+        background-color: #f1c40f;
+}
+.premium{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 125px;
+    width: 80vw;
+    padding: 5px;
+}
+.contain{
+    display:flex;
+   padding-left: 20%;
+   gap: 10px;
+}
+.box{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 300px;
+    width: 300px;
+    border: 1px solid black;
+
+    
+}
```

Comment:
Please check and remove white spaces and empty lines from code

Reply:
- [ ] Please when you suggest a change to the student make sure to give an explanation, a "why" you suggest that change 😉 😉
- [ ] Also use a couple of emojis and/or gifs in your review, they make the code review more human-friendly 👽🤖.